test/e2etest/installer_test.go

```diff
@@ -27,10 +27,19 @@ func TestInstaller(t *testing.T) {
 
 	// Tests come here
 
+	// Test the number of online remote agents
+	assertRemoteAgentList(t, testConfig)
+
 	// Uninstall and cleanup the environment
 	runUninstallScript(testConfig.ConfigPath)
 }
 
+func assertRemoteAgentList(t *testing.T, testConfig TestConfig) {
+	agentUrl := "admin/agent/configureAgents!doDefault.action"
+	url := fmt.Sprintf("https://%s@%s.%s.%s/%s", credential, product, testConfig.EnvironmentName, domain, agentUrl)
+	content := fmt.Sprintf("%s", GetPageContent(t, url))
```

Comment:
Same as https://github.com/atlassian-labs/data-center-terraform/pull/103#discussion_r786342371

Reply:
replied in #103